pkg/manifest/manifest_loader.go

```diff
@@ -248,39 +221,65 @@ func parseOAuth(a oAuth) (OAuth, error) {
 		return OAuth{}, fmt.Errorf("failed to parse ClientSecret: %w", err)
 	}
 
+	var urlDef UrlDefinition
+	if a.TokenEndpoint == nil {
+		urlDef = UrlDefinition{
+			Value: endpoints.Dynatrace.TokenURL,
```

Comment:
I think here you need to set `Type: Default` for the logic in the writer to work

Reply:
added